### ISSUE TITLE
Bugs with dindex

### ISSUE THREAD

**Issue by lintool** [castorini/pyserini#628]
hey @MXueguang - I'm running:

```
python -m pyserini.dindex --corpus integrations/resources/sample_collection_dense \
                          --encoder facebook/dpr-ctx_encoder-multiset-base \
                          --index index-tmp \
                          --batch 64 \
                          --device cpu \
                          --title-delimiter '\n'
```

The above works fine, but when I switch encoders:

```
python -m pyserini.dindex --corpus integrations/resources/sample_collection_dense \
                          --encoder castorini/ance-msmarco-doc-maxp \
                          --index index-tmp \
                          --batch 64 \
                          --device cpu \
                          --title-delimiter '\n'
```

I'm getting the following error:

```
Traceback (most recent call last):
  File "/anaconda3/envs/python36/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/anaconda3/envs/python36/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/jimmylin/workspace/pyserini/pyserini/dindex/__main__.py", line 54, in <module>
    model = init_encoder(args.encoder, device=args.device)
  File "/Users/jimmylin/workspace/pyserini/pyserini/dindex/__main__.py", line 33, in init_encoder
    return AnceDocumentEncoder(encoder)
  File "/Users/jimmylin/workspace/pyserini/pyserini/dindex/_base.py", line 80, in __init__
    self.model.to(self.device)
  File "/anaconda3/envs/python36/lib/python3.6/site-packages/torch/nn/modules/module.py", line 673, in to
    return self._apply(convert)
  File "/anaconda3/envs/python36/lib/python3.6/site-packages/torch/nn/modules/module.py", line 387, in _apply
    module._apply(fn)
  File "/anaconda3/envs/python36/lib/python3.6/site-packages/torch/nn/modules/module.py", line 387, in _apply
    module._apply(fn)
  File "/anaconda3/envs/python36/lib/python3.6/site-packages/torch/nn/modules/module.py", line 387, in _apply
    module._apply(fn)
  File "/anaconda3/envs/python36/lib/python3.6/site-packages/torch/nn/modules/module.py", line 409, in _apply
    param_applied = fn(param)
  File "/anaconda3/envs/python36/lib/python3.6/site-packages/torch/nn/modules/module.py", line 671, in convert
    return t.to(device, dtype if t.is_floating_point() or t.is_complex() else None, non_blocking)
  File "/anaconda3/envs/python36/lib/python3.6/site-packages/torch/cuda/__init__.py", line 164, in _lazy_init
    raise AssertionError("Torch not compiled with CUDA enabled")
AssertionError: Torch not compiled with CUDA enabled
```

What exactly is the "title-delimiter"? It's assuming a particular format of the input docs?

**Comment by lintool** [comment]
btw, @MXueguang - my intention was to compare with this: https://github.com/castorini/pyserini/pull/631

That is, reranking MS MARCO doc Indri baseline, comparing MaxP BM25 vs. MaxP ANCE.

**Comment by lintool** [comment]
Closed by #632
